src/commands/dnd/campaign/session/response.rs

```diff
@@ -37,3 +38,37 @@ pub async fn respond(
 
     responses::success(ctx, "Response recorded.").await
 }
+
+/// Responds to a D&D session (DM only), can respond for others
+#[poise::command(slash_command, check = "checks::dm_check")]
+pub async fn dmrespond(
+    ctx: Context<'_>,
+    #[description = "The ID of the session you're responding to"] session_id: i32,
+    #[description = "Who is going? Defaults to you."] respondee: Option<serenity::User>,
```

Comment:
```suggestion
    #[description = "Who is going?"] respondee: serenity::User,
```
We don't want the DM to respond to their own session. It's pretty much assumed that they're going since they are hosting it.

Reply:
[5605995](https://github.com/dotunwrap/dm-helper/pull/26/commits/560599526cb7babef49d00c0ecd2e2bd420e9e8c)